### DISCUSSION TITLE
Is it possible to export audio as wav?

### DISCUSSION THREAD

**Discussion by Kolaszg** [SteamDatabase/ValveResourceFormat#303] [General] [answered]
Hello, I was wondering if I could export audios as wav instead of mp3. I wanted to keep the exported audio intact, though since mp3 is lossy that's not really possible.

**Comment by xPaw** [comment, marked as answer]
It exports into the format that it is actually stored in, VRF does not do conversion (besides exporting).

**Reply by Kolaszg** [reply]
Oh, so the audio files are mp3? That's very odd, I'd expect at least ogg or something like that.
Thank you so much for replying by the way man.

**Reply by Sehyn** [reply]
Was looking for this too thanks.
Well lets go convertors.
